network/dag/notifier.go

```diff
@@ -291,7 +314,15 @@ func (p *notifier) Notify(event Event) {
 	}
 
 	if err := p.notifyNow(event); err != nil {
```

Comment:
🚨 The error is silently discarded now. Should at least do a `log.Print("dag: Notify retry on: ", err)`.

Reply:
It was only discarded when it triggered the retry mechanism. Changed to log all errors.